modules/installation-ibm-z-kvm-user-infra-machines-iso.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ibm_z/installing-ibm-z-kvm.adoc
+
+[id="installation-user-infra-machines-iso-ibm-z_kvm_{context}"]
+= Creating {op-system-first} machines
+
+Before you install a cluster on IBM Z infrastructure that you provision, you must install {op-system} as {op-system-base-full} guest virtual machines for the cluster to use. Complete the following steps to create the machines.
+
+.Prerequisites
+
+* At least 1 LPAR running {op-system-base} 8.3 with KVM, referred to as {op-system-base} KVM host in this procedure.
+* A domain name server (DNS) that can perform hostname and reverse lookup for the nodes. 
+
+.Procedure
+
+. Install the KVM/QEMU hypervisor on the {op-system-base} KVM host.
+
+. Obtain the {op-system-base} QCOW2 disk image file from the link:https://access.redhat.com/downloads/content/290[Product Downloads] page on the Red Hat customer portal or the link:https://mirror.openshift.com/pub/openshift-v4/s390x/dependencies/rhcos/4.2/latest/[{op-system} image mirror] page.
```

Comment:
Similar suggestion :) 
s/QCOW2/QEMU copy-on-write (QCOW2)

I know that QCOW2 is the updated version of QCOW, but I think the acronym means the same thing.